### PR TITLE
Update passthrough example and improve comments in HTTPS related examples

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: samples/ballerina-by-example/examples/https-service-invocation/https-service-invocation.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Run the service
+$ ballerina run https-service-consumption.bal -s
```
Comment (hasithaa) [inline samples/ballerina-by-example/examples/https-service-invocation/https-service-invocation.sh]: Correct command should be ` ballerina run **-s** https-service-consumption.bal`

File: samples/ballerina-by-example/examples/passthrough/passthrough.bal [hunk 2]
```diff
@@ -2,26 +2,26 @@ import ballerina.net.http;
 
 service<http> passthrough {
 
-    @Description {value:"Requests which contain any HTTP method will be directed to passthrough resource."}
+    endpoint<http:HttpClient> endPoint {
+        create http:HttpClient("http://localhost:9090/echo", {});
+    }
+
+    @Description {value:"Any type of HTTP requests will be dispatchd to the passthrough resource."}
```
Comment (chamil321) [inline samples/ballerina-by-example/examples/passthrough/passthrough.bal]: dispatched*
"Any type of HTTP requests"  doesn't sound meaningful to me as we need to emphasize on HTTP method specifically.

File: samples/ballerina-by-example/examples/https-services/https-services.client.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#Invoke the service using curl
+$ curl -k https://localhost:9095/hello
+Response: Hello World!
```
Comment (chamil321) [inline samples/ballerina-by-example/examples/https-services/https-services.client.sh]: End file with a new line

**Comment by pubudu91** [comment]
Closing this since this has become outdated

**Pull request closed without merging**